service/service.go

```diff
@@ -164,6 +164,12 @@ func New(config Config) (*Service, error) {
 			Logger:        config.Logger,
 
 			CertID: certs.ClusterOperatorAPICert,
+			// TODO: Reduce the max wait to reduce delay when processing
+			// broken tenant clusters.
+			//
+			// See https://github.com/giantswarm/giantswarm/issues/6703.
```

Comment:
I think the TODO issue should be put in some board. Do you have one? 

```suggestion
			//     https://github.com/giantswarm/giantswarm/issues/6703
```

Reply:
Yes I've added to the batman board and my own.

This is nicer but I copied `fmt` 😂  I'll update it!

https://github.com/giantswarm/fmt/blob/master/go/todo_comments.md